scss/prototypes/prototype-sample.html

```diff
@@ -0,0 +1,22 @@
+<div class="sample">
+	<h1>Sample prototype</h1>
+
+	<p>You only have a few requirements to setting up your sample prototype:</p>
+
+	<ul>
+		<li>Name it in the <code>prototypes</code> array in your huron.config.js.</li>
+		<li>Create a <code>prototypes</code> folder inside one of your KSS folders (that also should listed in your Huron config).</li>
+		<li>Name the prototype file should be <code>prototype-[name].html</code>.</li>
+		<li>Make sure to wrap the contents of the prototype with a single root element (e.g. <code>div</code>).</li>
+	</ul>
+
+	<hr>
+
+	<h2>Example KSS partials:</h2>
```

Comment:
We could probably direct people to the `huron` docs here as well, though probably more we could add in this repo in terms of best practices and ideas.